src/imio/esign/acroform.py

```
@@ -0,0 +1,192 @@
+        return u""
+    try:
+        if nbf.contentType == "application/pdf":
+            text = _pdf_text(nbf.data)
+        elif nbf.contentType in ZIPPED_XML_PARTS:
+            text = _zipped_xml_text(nbf.data, ZIPPED_XML_PARTS[nbf.contentType])
+        else:
+            return u""
+    except OSError:
+        logger.error("Cannot run '%s': the signature tags of pdf files are not checked!", PDFTOTEXT)
+        return u""
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)